cmd/bucket-quota.go

```diff
@@ -140,3 +146,33 @@ func enforceBucketQuotaHard(ctx context.Context, bucket string, size int64) erro
 	}
 	return globalBucketQuotaSys.enforceQuotaHard(ctx, bucket, size)
 }
+
+func enforceBucketThrottle(ctx context.Context, bucket string) error {
```

Comment:
This function must take the API handler calling this function, so that we can enforce quota throttle rules for the API under question alone.

Reply:
Ack. will change.